### ISSUE TITLE
PlanetScale cannot delete row "symbol myapp.`User`.id not found"

### ISSUE THREAD

**Issue by EvHaus** [prisma/prisma#13174]
### Bug description

I am using Prisma with PlanetScale and following the boilerplate for Next.js and next-auth as per https://github.com/planetscale/nextjs-planetscale-starter

When I go into Prisma Studio and I try to delete a user, I get this error:

```
Type: undefined
Message: 
Invalid `prisma.user.delete()` invocation:


  Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Server(ServerError { code: 1105, message: "symbol myapp.`User`.id not found", state: "HY000" })) })

Code: undefined

Query:
[object Object]
```

This only seems to happen when deleting from the User table. Deleting records from other tables seems fine. I am struggling to figure out how to parse this error message. I'm sure I did something wrong, but the error message doesn't give me much indication as to where the problem is.

Furthermore, connecting directly to the database at Planetscale and running `DELETE FROM User WHERE id = 'myid';` works fine. So I believe the issue is somewhere in Prisma.

### How to reproduce

1. Launch the boilerplate code from: https://github.com/planetscale/nextjs-planetscale-starter
2. Create a user
3. Launch Prisma studio
4. Try to delete the user

### Prisma information

```
datasource db {
    provider             = "mysql"
    url                  = env("DATABASE_URL")
    referentialIntegrity = "prisma"
}

generator client {
    provider        = "prisma-client-js"
    previewFeatures = ["referentialIntegrity"]
}

model Account {
    id                       String   @id @default(cuid())
    createdAt                DateTime @default(now())
    updatedAt                DateTime @updatedAt
    userId                   String
    type                     String
    provider                 String
    providerAccountId        String
    refresh_token            String?  @db.VarChar(500)
    access_token             String?  @db.VarChar(500)
    refresh_token_expires_in Int?
    expires_at               Int?
    token_type               String?
    scope                    String?
    id_token                 String?  @db.Text
    session_state            String?
    oauth_token_secret       String?
    oauth_token              String?
    user                     User     @relation(fields: [userId], references: [id], onDelete: Cascade)

    @@unique([provider, providerAccountId])
}

model Session {
    id           String   @id @default(cuid())
    sessionToken String   @unique
    expires      DateTime
    user         User?    @relation(fields: [userId], references: [id], onDelete: Cascade)
    userId       String?
}

model User {
    id            String               @id @default(cuid())
    createdAt     DateTime             @default(now())
    updatedAt     DateTime             @updatedAt
    name          String?
    email         String?              @unique
    password      String?
    emailVerified DateTime?
    image         String?
    accounts      Account[]
    sessions      Session[]
    organizations UserInOrganization[]
}

model VerificationToken {
    identifier String
    token      String   @unique
    expires    DateTime

    @@unique([identifier, token])
}

# These are some additional tables I've created in my app that are outside the boilerplate

model Organization {
    id         String               @id @default(cuid())
    createdAt  DateTime             @default(now())
    name       String               @unique
    users      UserInOrganization[]
    properties Property[]
}

model UserInOrganization {
    user           User         @relation(fields: [userId], references: [id], onDelete: Cascade, onUpdate: Cascade)
    userId         String
    organization   Organization @relation(fields: [organizationId], references: [id], onDelete: Cascade, onUpdate: Cascade)
    organizationId String
    assignedAt     DateTime     @default(now())
    assignedBy     String
    role           String       @default("member")

    @@id([userId, organizationId])
    @@index([userId])
    @@index([organizationId])
}

model Property {
    id                      String       @id @default(cuid())
    createdAt               DateTime     @default(now())
    name                    String
    organization            Organization @relation(fields: [organizationId], references: [id], onDelete: Cascade, onUpdate: Cascade)
    organizationId          String
    address                 String?
    number                  String?
    fileNumber              String?
    type                    String?
    numberOfUnits           Int?
    constructionYear        String?
    fiscalYearEnd           String?
    registeredAt            DateTime?
    comparableCityCPI       String?
    comparableCityInflation String?
    buildingClass           String?

    @@unique([organizationId, number])
}
```

### Environment & setup

- OS: Any
- Database: PlanetScale (MySQL)
- Node.js version: 17.8.0

### Prisma Version

```
3.13.0
```

**Comment by janpio** [comment]
This sounds like an instance of https://github.com/prisma/prisma/issues/10758 - which should be fixed soon. I am going to close this, let me know if you disagree.

**Comment by EvHaus** [comment]
Confirming that this is fixed in `prisma@3.14.0-dev.58`. Thank you!
